src/inc/LibraryIncludes.h

```diff
@@ -94,6 +94,7 @@
 
 // {fmt}, a C++20-compatible formatting library
 #include <fmt/format.h>

```

Comment:
Wait, if you're "replacing" it, we can't remove this line?

Reply:
This is the pre-roll include for _all projects in our entire solution_ ;) and it actually also is the main gateway into fmt.

There is no FMT_COMPILE without `fmt::format`, and `fmt::format` comes from `fmt/format.h`